modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -16,10 +16,41 @@ New features are added to the language continuously, and occasionally, some feat
 This section lists all of the features that have been removed, deprecated, added, or extended in different Cypher versions.
 Replacement syntax for deprecated and removed features are also indicated.
 
-
 [[cypher-deprecations-additions-removals-5.13]]
 == Version 5.13
 
+=== Updated features
+
+[cols="2", options="header"]
+|===
+| Feature
+| Details
+
+a|
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+SHOW FUNCTIONS YIELD *
+SHOW PROCEDURES YIELD *
+----
+a|
+Updated the `signatures` column in `SHOW FUNCTIONS`, and `SHOW PROCEDURES`.
+Procedure and function signatures now conform to the new Cypher Type syntax.
+
+Procedure signatures now follow the pattern:
+`"procedureName(param1 :: TYPE, param2 :: TYPE, .., paramN :: TYPE) :: (returnParam1 :: TYPE, returnParam2, .., returnParamN :: TYPE)"`
+
+With no return columns:
```

Comment:
```suggestion
The signature for procedures with no return columns now follows the pattern:
```

Maybe not correct - but in any case I think "With no return columns" needs to have more information.